apps/meteor/ee/app/livechat-enterprise/server/hooks/beforeNewRoom.js

```diff
@@ -10,20 +10,20 @@ callbacks.add(
 			return roomInfo;
 		}
 
-		const { priority: searchTerm } = extraData;
+		const { sla: searchTerm } = extraData;
 		if (!searchTerm) {
 			return roomInfo;
 		}
 
-		const priority = Promise.await(OmnichannelServiceLevelAgreements.findOneByIdOrName(searchTerm));
-		if (!priority) {
-			throw new Meteor.Error('error-invalid-priority', 'Invalid priority', {
+		const sla = Promise.await(OmnichannelServiceLevelAgreements.findOneByIdOrName(searchTerm));
+		if (!sla) {
+			throw new Meteor.Error('error-invalid-priority', 'Invalid sla', {
```

Comment:
```suggestion
			throw new Meteor.Error('error-invalid-sla', 'Invalid sla', {
```

Reply:
oops, it was wrong in a few other places too